runtest-moduleapi

```diff
@@ -1,18 +1,19 @@
 #!/bin/sh
```

Comment:
This file seems to have been committed with CRLF endings, leading to "/bin/sh^M: bad interpreter" errors. This can be fixed by running `set fileformat=unix` in vim and saving.